### DISCUSSION TITLE
Daily 10/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#180] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Ayer me puse al dia con todos los cuestionarios y me dediqué a leer todo el material del ispc, me puse al dia con testing y ciberseguridad. Estuve viendo los conceptos tipos de capas de datos, junto a pruebas unitarias y demas resumiendolos para comprender mejor.
Me estoy poniendo al dia con otras clases fuera del ispc.
Esta noche me voy asignar un par de issues, voy avanzar con el back y el front en el proyecto.

**Comment by erick-lpz** [comment]
Ayer estuve completando las tareas del ispc y viendo el material nuevo para poner en practica en el proyecto.
Hoy voy a crear la activity de ver usuarios en el dashboard_admin

**Comment by MarielaGomez** [comment]
Ayer estuve repasando la conexión con base de datos y con el material subido en plataforma, también revisar los activitys y mejorar la comunicación con los botones

**Comment by NataliaAlvarezIspc** [comment]
Ayer completé todos los cuestionarios de App Móviles de la plataforma, también estuve leyendo un poco de los materiales que se encuentan ahí.
Hoy comenzaré con la tarea asignada de registración. Tengo que instalarme el IntelliJ y veré como funciona todo, ya que mi computadora no cuenta con muchos recurso y ya está casi al límite.

**Comment by marceloedelgado** [comment]
Ayer y hoy estuve abocado a completar las actividades y cuestionarios de las materias en la plataforma del ISPC. Hoy estuve viendo sobre el CRUD y la conexión a la base de datos.
